Kitodo/src/main/java/de/sub/goobi/forms/AktuelleSchritteForm.java

```diff
@@ -852,6 +852,25 @@ public void setMySchritt(Task task) {
         setStep(task);
     }
 
+    /**
+     * Get task with specific id.
+     *
+     * @param id
+     *            passed as int
+     * @return task
+     */
+    public Task getTaskById(int id) {
+        try {
+            return serviceManager.getTaskService().getById(id);
+        } catch (DAOException e) {
+            logger.error(e.getMessage());
+            FacesContext facesContext = FacesContext.getCurrentInstance();
```

Comment:
Why you are introducing a new way to report error messages to the user? Even this message is not translatable.

Reply:
This is the Java bean alternative to rendering messages to templates that have `h:message` or `h:messages` tags. It should therefore be in line with the existing error reporting in templates like `editDocket.xhtml` and others.